### ISSUE TITLE
ACE interact keybinding should re-enable controls - currently does nothing

### ISSUE THREAD

**Issue by Saborknight** [Saborknight/9liners-and-notepad#37]

**Comment by TheMysteriousVincent** [comment]
What was meant by that title?

**Comment by Saborknight** [comment]
@TheMysteriousVincent I have no idea why this didn't have a description before. My apologies, this has been updated now. 

So yeah, I think the sheet should disappear completely, since we're assuming that this would be done in an emergency, or when someone's done with the sheet... I think. Although, I'm still on the fence about this... since on the other hand, we might, in the future, implement some ACE interactions when the sheet is being edited... so in that case it may be a good idea to prepare the interface for that so that users aren't being forced to change what they are used to... Also, the ESC key should be hiding the sheets anyway (Is it thought? I actually can't remember).

**Comment by TheMysteriousVincent** [comment]
I agree on the emergency button escape. That shouldnt be any problem at all.

The Escape-Key works fine because thats the default ArmA 3 button to escape Dialogs.

**Comment by Saborknight** [comment]
Through speaking on the ACE3 Slack, I've been made aware that the ACE Interact key will disable a dialog, but not a display. However, the dialog is exactly what this issue is about... this indicates that the dialog has something wrong/unpredicted in its definition.
